### PR TITLE
Scroll to the top result when stream is updated

### PR DESCRIPTION
**Pull request by pe4cey**

### CODE REVIEW

File: src/browser/modules/Stream/Stream.jsx [hunk 1]
```diff
@@ -20,6 +20,7 @@
 
 import { connect } from 'preact-redux'
 import { Component } from 'preact'
+import ReactDOM from 'react-dom'
```
Comment (FylmTM) [inline src/browser/modules/Stream/Stream.jsx]: Is react-dom in package.json?
Looks like that it's not there.

**Comment by FylmTM** [comment]
Just a side question - we can use react-dom?

I thought adding it to enable testing via enzyme. But after conversation I found out that React was abandoned due to license & patents issues.
And unfortuntately react-dom is under same BSD license.

**Comment by oskarhane** [comment]
No, we should not use react-dom but I suspect that this works because it's gets aliased to preact-compat that has the findDomNode.
But it would be better to be more clear and import the aliased method directly.

**Comment by pe4cey** [comment]
Now preactified

**Pull request merged**